packages/admin-ui/src/pages/community/components/TreasuryActions.tsx

```diff
@@ -1,14 +1,11 @@
 import Button from "components/Button";
 import React from "react";
-import { mediumTreasuryUrl, explorerTreasuryUrl } from "params";
+import {explorerTreasuryUrl} from "params";
 import newTabProps from "utils/newTabProps";
 
 export default function TreasuryActions() {
   return (
     <>
-      <Button href={mediumTreasuryUrl} {...newTabProps}>
```

Comment:
Why removing the treasury?

Reply:
Discus during/after daily? Not worth to write it out right now.

Reply:
agreed we should probably just remove the entire section for the treasury, link already removed is way outdated and doesnt work like that, the remaining explorer link does work ans shows the sourcecred dashboard to show stats, but it doesnt pay. out, hasnt since April, and it doesnt allow new applicants on discord since the channels and bots were removed/shut disabled for months now since the last revamp.  we have a new way to track all user contributions with the Statbot on discord and I think is a far better thing to do now no need to deal with the #ct-addresses channel doesnt even exist anymore and just message the largest PAN holders to  withdraw now if they so desire, PAN fell massively in value as well. making it much less relevant, let me know if you agree to just remove the whole section and ill add those commits here